src/features/MovieList/MovieTile/styled.js

```diff
@@ -66,27 +68,31 @@ export const Year = styled.p`
     }
 `;
 
-export const TagsContainer = styled.div`
+export const TagsContainer = styled.ul`
     margin-top: 8px;
     display: flex;
     flex-wrap: wrap;
     gap: 8px;
+    padding: 0;
+    list-style: none;
+    margin-bottom: 0;
```

Comment:
you could use `margin: 8px 0 0;`

Reply:
margin: 8px 0 0;         is more optimalized than  margin-top ? 
i can changed this but what purpose ?

Reply:
You replaced only margin-top with `margin: 8px 0 0;` what i meant you should do is replace margin-top and margin-bottom with one margin shorthand. Using shorthands is very convenient and in my opinion specific attributes like margin-top should be used only if we want to overwrite just one property. This way reading code is easier and more elegant. It is purely esthetics and time-saving matter just as not specifying units after setting property as 0. Importance of this is arguable since it provides next to none optimalization improvement, however if you ask me it is not only about if the solution is working but also if it is intuitive and coherent.

Reply:
I didn't noticed I also have a margin-bottom there, sorry, fixed.